.github/workflows/build-docker-image-and-binaries.yaml

```
@@ -24,11 +24,20 @@ jobs:
       id-token: write
 
     steps:
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+
       - name: Validate tag
         env:
           SEMVER: ${{ github.event.inputs.tag }}
         run: |
           if [[ $SEMVER =~ v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3} ]]; then echo "Tag is okay" && exit 0; else echo "invalid tag" && exit 1; fi
+          aws s3 ls s3://axelar-releases/axelard/$SEMVER && echo "tag already exists, use a new one" && exit 1          
```

```suggestion
          aws s3 ls s3://axelar-releases/axelard/"$SEMVER" && echo "tag already exists, use a new one" && exit 1          
```
Perhaps quote the shell variable expansion?

---

It will work in both cases. 

kalidax@debian:~$ SEMVER=v0.14.1
kalidax@debian:~$ aws s3 ls s3://axelar-releases/axelard/"$SEMVER"
                           PRE v0.14.1/

kalidax@debian:~$ aws s3 ls s3://axelar-releases/axelard/$SEMVER
                           PRE v0.14.1/

But you are right, it is a best practice with quotes.